compare/src/components/atoms/ImageScrubber.js

```diff
@@ -99,7 +95,7 @@ export default class ImageScrubber extends React.Component {
 
     const scrubberTestImageSlug = this.props[testImageType];
 
-    function getDiverged (arg) {
+    function getDiverged (arg) { // eslint-disable-line
```

Comment:
This function is unused but I assume it will be used as part of the new diverged work.

Reply:
Correct